src/runner/task.js

```diff
@@ -74,6 +75,32 @@ export default class Task extends AsyncEventEmitter {
 
     }
 
+    _prepareTestStructure (tests) {
+        const groups = groupBy(tests, 'fixture.id');
+
+        return Object.keys(groups).map(fixtureId => {
+            const testsByGroup = groups[fixtureId];
+            const fixture      = testsByGroup[0].fixture;
+
+            return {
+                fixture: {
+                    id:      fixture.id,
+                    name:    fixture.name,
+                    pageUrl: fixture.pageUrl,
+                    path:    fixture.path,
+                    tests:   testsByGroup.map(test => {
+                        return {
+                            id:      test.id,
+                            name:    test.name,
+                            pageUrl: test.pageUrl,
+                            skip:    test.skip
```

Comment:
I see we add to report the `skip` property. 
But both fixture and test entities have `skip` and `only` properties. Should we add this information to the report?

@aleks-pro

Reply:
At this moment we have quite unclear approach to detect if we should skip or execute `skip/only` tests. I would prefer to remove the `skip` property at all, if it is not required for @aleks-pro

Reply:
It should suit our needs if you leave it as it is